### DISCUSSION TITLE
Using SQLclchemy,How to create a Postgresql partitioned table

### DISCUSSION THREAD

**Discussion by YuexiaW** [sqlalchemy/sqlalchemy#11980] [Usage Questions]
Using SQLclchemy,How to create a Postgresql partitioned table,And you can use alembic to migrate

**Comment by CaselIT** [comment]
Hi, 

it's documented here https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#postgresql-table-options

**Comment by YueXiaWyl** [comment]
thank you very much
How to create subtables of a partitioned table，I intend to use sqlchemy to create subtables of the partition table
